src/finch/processes/ensemble_utils.py

```diff
@@ -356,6 +356,7 @@ def make_ensemble(  # noqa: D103
     files: list[Path],
     percentiles: list[int],
     spatavg: bool | None = False,
+    tmpavg: bool | None = False,
```

Comment:
```suggestion
    tmpavg: bool  = False,
```
Nit pick, but I don't think there's a way to pass None here, and it wouldn't mean anything.